src/telemetry/logs.ts

```diff
@@ -172,12 +166,13 @@ class OTLPLogQueueTransport extends TransportStream {
   readonly applicationID: string;
   readonly executorID: string;
 
-  constructor(readonly telemetryCollector: TelemetryCollector) {
+  constructor(readonly telemetryCollector: TelemetryCollector, logLevel: string) {
     super();
+    this.level = logLevel;
```

Comment:
This sets the desired log level to the parent `TransportStream` class. `Winston` decides which event to dispatch to which transport based on what level they are listening to.

Reply:
Nice fix!